src/components/navbar/mobile/parts/profileDataM/ProfilePicAndNameNavM.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { useStore } from '@nanostores/react';
+import { storeProfileData } from '@src/components/profile/stores/store-profile-data';
+
+const ProfilePicAndNameNavM = () => {
+  const { data, loading } = useStore(storeProfileData);
```

Comment:
Use storeLoggedUser here storeProfileData is only defined on /profile

Reply:
also when using the right store the sidepanel is transparent when open
![image](https://github.com/NavigoLearn/Navigo-FrontEnd/assets/32602702/844e4510-259e-47f1-8737-ab757ff0f080)